src/IonSpan.ts

```diff
@@ -51,6 +51,14 @@ export abstract class Span {
 
   protected abstract clone(start: number, len: number): Span;
 
+  locator(pos: number) : number[] {
+    throw new Error("not implemented");
```

Comment:
So are you OK with my fixing makeReader so that it creates the correct Span type, changing the text reader and raw parser to accept StringSpan, and moving these methods out of the abstract Span class into StringSpan?